bin/doc/json.js

```diff
@@ -486,7 +485,7 @@ const finishSection = (section, parent) => {
 // Not a general purpose deep copy.
 // But sufficient for these basic things.
 const deepCopy = (src, dest) => {
-  Object.keys(src).filter((k) => !dest.hasOwnProperty(k)).forEach((k) => {
+  Object.keys(src).filter((k) => !Object.prototype.hasOwnProperty.call(k)).forEach((k) => {
```

Comment:
```suggestion
  Object.keys(src).filter((k) => !Object.prototype.hasOwnProperty.call(dest, k)).forEach((k) => {
```

Reply:
Wow that was a big one to miss..  Can't beat 11pm sloppy work 🖌️